packages/core/src/common/object.spec.ts

```diff
@@ -0,0 +1,111 @@
+// *****************************************************************************
```

Comment:
Given that the file is `objects.ts` the file should be renamed to `objects.spec.ts` (notice the missing `s`).